kernel/kernel-api/src/test/scala/io/delta/kernel/test/MockTableClientUtils.scala

```diff
@@ -70,12 +70,36 @@ trait MockTableClientUtils {
           throw new UnsupportedOperationException("not supported in this test suite"))
     }
   }
+
+  def longVector(values: Long*): ColumnVector = new ColumnVector {
```

Comment:
can you move the to `VectorTestUtils`? Currently this file is in defaults module. Move it to the API module and add these helper methods there.

Reply:
Moved.